Update index.rst (#90)
scikit-image/skimage-web: index.rst

News
----

> - **Release!** Version 0.21.0 2023-06-02
> - **Release!** Version 0.20.0 2023-02-28

Reviewer: ```suggestion
- **Release!** `Version 0.21.0 <https://scikit-image.org/docs/stable/release_notes/release_0.21.html>`_ 2023-06-02
- **Release!** `Version 0.20.0 <https://scikit-image.org/docs/0.20.x/release_notes.html>`_ 2023-02-28
```

> - `scikit-image selected for third round of funding by CZI
>   <https://chanzuckerberg.com/eoss/proposals/?cycle=5>`__ 2022-11-30

Reviewer: ```suggestion
- `scikit-image selected for third round of funding by CZI
  <https://chanzuckerberg.com/eoss/proposals/from-library-to-protocol-scikit-image-as-an-api-reference/>`__ 2022-11-30
```

Reviewer 2: I think two is technically correct.

Author: > I think two is technically correct.

Yes! @lagru I didn't know either until [this conversation](https://github.com/scikit-image/scikit-image/pull/6497#discussion_r999679301) took place, where I learned from @stefanv that the double underscore is actually more correct.

Reviewer: Wow, today I learned. Good to know going forward and thanks for the link.

Reviewer: Note, I updated the suggestion accordingly.

> - `scikit-image selected for third round of funding by CZI

Reviewer 2: Maybe we can be a bit more explicit here: "Funding received for X as part of CZI's 5-th EOSS cycle"

Author: Ok. What about:
"As part of CZI's 5-th EOSS grant cycle, scikit-image received funding for creating a typed, discoverable, and extensible API!"
?

Reviewer 2: That sounds great!

News
----

- **Release!** `Version 0.21.0 <https://scikit-image.org/docs/stable/release_notes/release_0.21.html>`__ 2023-06-02
- **Release!** `Version 0.20.0 <https://scikit-image.org/docs/0.20.x/release_notes.html>`__ 2023-02-28

> - As part of CZI's 5th EOSS grant cycle, scikit-image received funding for
>   `creating a typed, discoverable, and extensible API

Reviewer: Actually, to me this sounds better... :smile: 
```suggestion
- As part of CZI's 5th EOSS grant cycle, scikit-image received funding to
  `create a typed, discoverable, and extensible API
```
